docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -345,17 +347,4 @@ or you can specify the scenario or pipeline by passing it as a parameter.
 
     This example will produce the following output:
 
-    ```
-    Submit: scenario_1
-    my_global_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.
-    my_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.
-    my_subscriber_multi_param: params ["my_param_1", 42]; task 'my_task_1 .
-    my_subscriber: scenario 'my_scenario_1' ; task 'my_task_2'.
-    my_subscriber_multi_param: params ["my_param_1", 42]; task 'my_task_2'.
-    Submit: scenario_2
-    my_global_subscriber: scenario 'my_scenario_2'; task 'my_task_1'.
-    Unsubscribe to my_global_subscriber for scenario_1
-    Submit: scenario_1
-    my_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.
-    my_subscriber: scenario 'my_scenario_1'; task 'my_task_2'.
-    ```
+    ``  Submit: scenario_1     my_global_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.     my_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.     my_subscriber_multi_param: params ["my_param_1", 42]; task 'my_task_1 .     my_subscriber: scenario 'my_scenario_1' ; task 'my_task_2'.     my_subscriber_multi_param: params ["my_param_1", 42]; task 'my_task_2'.     Submit: scenario_2     my_global_subscriber: scenario 'my_scenario_2'; task 'my_task_1'.     Unsubscribe to my_global_subscriber for scenario_1     Submit: scenario_1     my_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.     my_subscriber: scenario 'my_scenario_1'; task 'my_task_2'.  ``
```

Comment:
```suggestion
    ```
    Submit: scenario_1
    my_global_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.
    my_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.
    my_subscriber_multi_param: params ["my_param_1", 42]; task 'my_task_1 .
    my_subscriber: scenario 'my_scenario_1' ; task 'my_task_2'.
    my_subscriber_multi_param: params ["my_param_1", 42]; task 'my_task_2'.
    Submit: scenario_2
    my_global_subscriber: scenario 'my_scenario_2'; task 'my_task_1'.
    Unsubscribe to my_global_subscriber for scenario_1
    Submit: scenario_1
    my_subscriber: scenario 'my_scenario_1'; task 'my_task_1'.
    my_subscriber: scenario 'my_scenario_1'; task 'my_task_2'.
    ```
```